compiler/expr_visitor_test.py

```diff
@@ -75,13 +75,10 @@ class Foo(object):
   testBinOpArithmeticMod = _MakeExprTest('9 % 5')
   testBinOpArithmeticMul = _MakeExprTest('3 * 2')
   testBinOpArithmeticOr = _MakeExprTest('2 | 6')
+  testBinOpArithmeticPow = _MakeExprTest('2 ** 16')
```

Comment:
A couple more test cases would be good here, e.g. negative numbers, floats, etc.